### DISCUSSION TITLE
Can you unlock DLCS with Peacock?

### DISCUSSION THREAD

**Discussion by Undore** [thepeacockproject/Peacock#478] [Q&A] [answered]
Its possible to unlock all suits, including deluxe ones. Is it possible to unlock all dlcs then?

**Comment by AnthonyFuller** [comment, marked as answer]
Nevermind! Apologies for my last response, I misread. No, to unlock DLC content, you must pay for the DLCs.

**Comment by FooYou2497** [comment]
your question is like telling everyone:
Hi, IO, is it okay to steal from your company?

then you can only get one answer:
No.
